include/oneapi/dpl/internal/version_impl.h

```diff
@@ -0,0 +1,40 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_VERSION_IMPL_H
+#define _ONEDPL_VERSION_IMPL_H
+
+#if _ONEDPL___cplusplus >= 202002L && __has_include(<version>)
+#    include <version> // The standard C++20 header
+#    define _ONEDPL_STD_FEATURE_MACROS_PRESENT 1
```

Comment:
In the file https://github.com/oneapi-src/oneDPL/blob/0414309eccb641e5e250af215e904cedead9f2f8/include/oneapi/dpl/pstl/onedpl_config.h#L32
we have the code:
```C++
#if _ONEDPL___cplusplus >= 202002L && __has_include(<version>)
#    include <version>
#    define _ONEDPL_STD_FEATURE_MACROS_PRESENT 1
#endif
```

Should we add `#else` branch there?

Reply:
Or may be better to use `_ONEDPL_STD_FEATURE_MACROS_PRESENT` in tests ?

Reply:
I would leave the tests as they are, at least until there is a need to explicitly define the macro to 0.

Reply:
ok